src/common/enums.js

```diff
@@ -141,22 +141,26 @@ export const fileTypeDetail = {
   SHM: {
     icon: 'shipping',
     title: 'Shipping Manifest',
-    description: 'File type description goes here...',
+    description:
+      'Manifest of biospecimens shipped to the sequencing center. The biospecimen identifiers from this file will be used to link all clinical and genomic data in the KFDRC platform.',
```

Comment:
"link all clinical and genomic data" - this isn't really true. I think this is being mixed up with the sequencing manifest. The sequencing manifest is what links genomic files to the specimens they came from, thereby linking all clinical and genomic data.

Reply:
I think perhaps "to account for" vs "to link"?